### PR TITLE
fix: usage of `node12 which is deprecated` in CI

### LINKED ISSUE #692: `audit.yml` uses `actions-rs/install@v0.1` which `uses node12 which is deprecated`
https://github.com/FyroxEngine/Fyrox/actions/runs/11415918507

### PR DESCRIPTION
**Pull request by hamirmahal**
fixes #692.

### CODE REVIEW

**Review by hamirmahal — COMMENTED** [review]
[The `audit` workflow, which this pull request modifies, passes, with the warning about `node12` gone](https://github.com/FyroxEngine/Fyrox/actions/runs/11421648777?pr=693).

File: .github/workflows/audit.yml [hunk 1]
```diff
@@ -23,15 +23,6 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      # Actions-rs/install is unmaintained but works for now.
-      # Alternatives for when it breaks:
-      #  - dtolnay/install - not a full replacement - only supports crates he uses (not audit currently)
-      #  - baptiste0928/cargo-install - looks like it just compiles the crate and then caches it for subsequent runs
-      #  - just `cargo install` and caching it
-      - uses: actions-rs/install@v0.1
```
Comment (mrDIMAS) [inline .github/workflows/audit.yml:31]: This change seems to remove installation of `cargo-audit` entirely?

Comment (hamirmahal) [reply .github/workflows/audit.yml:31]: That's correct.

Comment (hamirmahal) [reply .github/workflows/audit.yml:31]: It appears to be unnecessary. Maybe the GitHub runners come with it?

**Comment by mrDIMAS** [comment]
I removed cargo audit entirely because it is useless, this PR now has no effect so can be closed.

**Pull request closed without merging**